### PR TITLE
76 uw 113

### PR DESCRIPTION
**Pull request by ryanlong1004**
https://jira-epic.woc.noaa.gov/browse/UW-113

### CODE REVIEW

File: src/uwtools/files/model/file.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    @property
+    def exists(self):
+        """returns true if the file exists"""
+        return os.path.exists(self.path)
+
+    @property
+    def dir(self) -> List[Any]:
+        """returns the contents of the directory recursively"""
+        raise NotImplementedError
```
Comment (christinaholtNOAA) [inline src/uwtools/files/model/file.py:97]: Will these un-implemented methods have corresponding Issues opened, or are you already working on their implementation?

These could potentially be a great space for Andrew to hop in.

File: tests/files/test_file_manager.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    destination = Unix(f"file://{temp_dir}")
+
+    fm: UnixFileManager = FileManager.get_file_manager(Prefixes.UNIX)
+    fm.copy([source], [destination.path])
+
+    actual = len(os.listdir(temp_dir))
+    expected = 6
+    assert actual == expected
+
+    # assert os.path.exists()
```
Comment (christinaholtNOAA) [inline tests/files/test_file_manager.py:57]: Maybe clean up this line.

**Review by christinaholtNOAA — COMMENTED** [review]
This looks good. Just a couple of questions and minor comments.

**Comment by ryanlong1004** [comment]
Are you talking about the unit test fixtures?

**Comment by ryanlong1004** [comment]
> > @aerorahul are you talking about the unit test fixtures?
> 
> ```
> fixtures/files/1.file
> fixtures/files/2.file
> fixtures/files/3.file
> ```

How do you suggest I create tests?

File: src/uwtools/files/model/file.py (hunk 1 above)
Comment (ryanlong1004) [reply src/uwtools/files/model/file.py:97]: That's up to stakeholders/product owner.

Otherwise I can remove them.

**Comment by aerorahul** [comment]
> > > @aerorahul are you talking about the unit test fixtures?
> > 
> > 
> > ```
> > fixtures/files/1.file
> > fixtures/files/2.file
> > fixtures/files/3.file
> > ```
> 
> How do you suggest I create tests?

I don't think you need 50 mb files for the tests.
You could create a few ascii files.

File: src/uwtools/files/model/file.py (hunk 1 above)
Comment (christinaholtNOAA) [reply src/uwtools/files/model/file.py:97]: I'm totally fine leaving the code as-is. Just curious if there was (or will be) a corresponding Issue.

**Review by christinaholtNOAA — APPROVED** [review]

**Pull request merged**